diff --git a/lib/index.d.ts b/lib/index.d.ts
index 04b79d64668de7a55d802ce8a5aca705fd683a9f..20296dac6cca3da320b4cd18ced001118465f585 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -14,6 +14,6 @@
 export type { StyleCache } from './getStyleCache.js';
 export type { CacheObject, EventHandlerKeys } from './types.js';
 export type { CommonComponentProp } from './parseStyleProps.js';
-export type { CSSProperties, JsxstyleComponentStyleProps } from './types.js';
+export type { CSSProperties, JsxstyleComponentStyleProps, PseudoPrefixedProps } from './types.js';
 export type { MakeCustomPropertiesFunction, BuiltCustomProperties, VariantOptions, } from './makeCustomProperties.js';
 export type { VariantMap, CustomPropsObject, CustomPropValuesObject, GetCustomProperties, } from './generateCustomPropertiesFromVariants.js';